src/util/forge-config.js

```diff
@@ -37,6 +38,17 @@ const proxify = (object, envPrefix) => {
   });
 };
 
+/**
+ * Sets sensible defaults for the `config.forge` object.
+ */
+export function adjustForgeConfig(packageJSON) {
```

Comment:
maybe `s/adjustForgeConfig/setInitialForgeConfig`?

Reply:
Yeah, that's better.

Reply:
Renamed.